content/departments/support/process/p4-enablement.md

```diff
@@ -190,6 +190,64 @@ pass12349ers
 
 (Note the value listed is from the dockerfile arg declaration)
 
-### GUI tools
+## GUI tools
 
 You can download the [Helix Admin Tool](https://www.perforce.com/downloads/administration-tool) which makes it easier to configure users, permissions, etc.
+
+## Testing sub-repo permissions
+
+Sub-repo permissions are still experimental and below are the steps required to start testing them locally against our dogfood perforce server.
+
+Add this configuration to your external service configuration in dev private
+
+```yaml
+"PERFORCE": [
+    {
+      "p4.port": "perforce.sgdev.org:1666",
+      "p4.user": "admin",
+      "p4.passwd": "REDACTED",
+      "depots": {
+        "//test-perms/"
+      ],
```

Comment:
Should this be a `}` instead? I'm just trying to get this to work for me and getting json errors

Reply:
Oh wait actually I think the open bracket is the one that needs to change to `[`